### ISSUE TITLE
Get rid of Google Calendar

### ISSUE THREAD

**Issue by jbelien** [oSoc19/webapp-wip#42]
Let's try to find a replacement for the Google Calendar in the "Practical > When" page.

I'm sure it's possible to find a "better looking" and open solution for the calendar !

**Comment by jbelien** [comment]
Now that we use Nexcloud, it's **really** easy to replace the Google Calendar by Nextcloud Calendar !

New agenda > Share > + > Copy embedding code

**Comment by mietcls** [comment]
Good point, if we can find one that people can subscribe to that would be awesome ^^

**Comment by jbelien** [comment]
> Good point, if we can find one that people can subscribe to that would be awesome ^^

**Nextcloud calendar !**
We **already** have **everything** we need!

https://docs.nextcloud.com/server/latest/user_manual/en/pim/calendar.html
